File: module_wan/src/app/java/com/huyingbao/module/wan/WanApplication.java
package com.huyingbao.module.wan;

import com.huyingbao.core.common.CommonApp;

import dagger.android.AndroidInjector;
import dagger.android.support.DaggerApplication;

/**
 * Created by liujunfeng on 2019/1/1.
 */
public class WanApplication extends CommonApp {
    @Override
    protected AndroidInjector<? extends DaggerApplication> applicationInjector() {
        return DaggerWanComponent.builder().application(this).build();
    }
}

File: module-wan-kotlin/src/app/java/com/huyingbao/module/wan/kotlin/WanApplication.kt
package com.huyingbao.module.wan.kotlin


import com.huyingbao.core.annotations.RxAppBody
import com.huyingbao.core.base.BaseApp
import dagger.android.AndroidInjector
import dagger.android.support.DaggerApplication
import org.greenrobot.eventbus.EventBus

/**
 * Created by liujunfeng on 2019/1/1.
 */
@RxAppBody
class WanApplication : BaseApp() {
    override fun applicationInjector(): AndroidInjector<out DaggerApplication> {
        return DaggerWanComponent.builder().application(this).build()
    }
}
